utils/regression-analysis/src/main.rs

```diff
@@ -110,11 +110,11 @@ struct GithubActionBenchmark {
 
 #[derive(Deserialize)]
 #[serde(transparent)]
-struct WeightBenchmark(Vec<Weight>);
+struct WeightBenchmark(Vec<u64>);
```

Comment:
How it will deserialise u64 if Weight type used for all benches?

Reply:
It will use JSON file generated using our `regression-analysis-weight-template.hbs`